js/colourpicker.js

```diff
@@ -369,129 +386,209 @@
 
       }
 
-      var hue, saturation, brightness, x, y, r, phi,
-
-      hex = input.val(),
+      var hue, saturation, brightness, x, y, r, phi, rgb, hsb, alpha, hex,
 
       // Helpful references
-      colourpicker = input.parent(),
+      colourpicker = input.parent();
       settings = input.data('colourpicker-settings'),
 
       // Panel objects
       grid = colourpicker.find('.colourpicker-grid'),
+      nnergrid = colourpicker.find('.colourpicker-grid-inner'),
```

Comment:
though honestly, I should probably go back and change the container class situation back to the way you had it originally: since we nixed the checkered background on the grid itself there is no longer any need for two inner containers on the grid.